### ISSUE TITLE
FATAL: password authentication failed for user "axelor"

### ISSUE THREAD

**Issue by pzehle** [axelor/axelor-docker#6]
In your Docker [installation documentation](https://docs.axelor.com/abs/5.0/install/docker.html), you state: 

> Following environment variables can be used to change container settings:

So I created the container changing the `POSTGRES_PASSWORD` variable, to something secure, because of course the database being secured by the same name as the user is not secure at all. Well, it doesn't work.
I've been here for hours and hours. If I don't change this variable on creation, all works, but if I do, it crashes. I tried changing the `application.properties` and `app.properties` variables located at `/var/lib/tomcat`, and also changing the variables located directly in the ROOT folder. Nothing changes and it is destroying me. Also tried directly changing the password of the user "axelor" directly on the psql cli.

Help please!

**UPDATE:**
Seems that the ENV variable is only working to change the password in PSQL, because if I change the password back to _axelor_ it works. So this variable is not changing the information in the system files.

**Comment by pzehle** [comment]
So, after more hours of looking into this issue, I saw that `app.properties` located under `/var/lib/tomcat/` has the same configuration variables but **at the end of the file**. This variables are the ones that need to be changed.

This issue can now be closed and hope that if anyone is going through this can find this useful. Thanks anyway.

**Comment by maovidal** [comment]
Dealing with the application.properties file has been very confusing to me.

The multiple references and appearances of this file on the generated aio-base and aio-erp containers, exacerbated my frustration building the images, mounting volumes to trace why the configuration was not applied and trying to understand the entrypoint.sh scripts. 

Still I don't understand why happens that duplicated entry for the database configuration on the app.properties. Without @pzehle finding I would be still struggling and ready to give up.

For reference, I've managed to deploy the current 5.3 WAR using aio-base (not aio-erp per issue #2) as it allowed me to mount the tomcat folder on the host system to easily change the app.properties file.

**Comment by pzehle** [comment]
@MrMauro yes, this has been hard for me too. I still haven't found configurations I need to change, sometimes I change the application.properties, some other times I change the app.properties and finally changing application.properties under `ROOT` makes every time a different result. This is very confusing, but still, installing with Docker was the best choice, as the poor documentation on how to install manually results in 0 chance to set up the application properly.

I am glad I could help with something. I will be posting a lot of issues of things I am encountering, and somehow I will try to help document the application, as it is very hard to use.

**Comment by maovidal** [comment]
Thank you Paulo. I'm rooting for this project as it seems like a great option for me compared to other alternatives out there. I'm also willing to contribute in any way I can.

So far I've managed to propose #7 which helped me to run the application with the aio-base container. I think it fixes the issue with the app.properties configuration at creation.

**Comment by bsikander** [comment]
Any update on this issue?
Not mounting volume to host is not a good solution.

**Comment by andrewschmidt-a** [comment]
I am still having issues configuring this docker image. Not sure this is actually working...
